.circleci/config.yml

```diff
@@ -296,7 +296,11 @@ jobs:
           path: ~/wheels
   check_release:
     docker:
-      - image: cimg/python:3.10
+      - image: girder/tox-and-node
+    parameters:
+      node:
+        type: string
+        default: v20
```

Comment:
@manthey I'm a bit confused about the failure of the `check_release` [job for this branch](https://app.circleci.com/pipelines/github/girder/large_image/6386/workflows/58cf8ea3-bed1-4cb1-81e8-859b6b832727/jobs/45523). I believe that `girder/tox-and-node` is built from the `test.Dockerfile` file, which [does install nvm](https://github.com/girder/large_image/blob/ec069cb0b1dc9bc113f04b96a2f12703c0a27c6d/test.Dockerfile#L119-L120). Am I misinterpreting the `docker.image` config option?